composer.json

```
@@ -17,16 +17,16 @@
         {"name": "Marco Pivetta", "email": "ocramius@gmail.com"}
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^7.1 || ^8.0",
         "ext-tokenizer": "*",
         "doctrine/annotations": "^1.0"
     },
     "require-dev": {
         "doctrine/coding-standard": "^6.0",
         "doctrine/common": "^2.10",
-        "phpstan/phpstan": "^0.11.0",
-        "phpstan/phpstan-phpunit": "^0.11.0",
-        "phpunit/phpunit": "^7.0"
+        "phpstan/phpstan": "^0.11.0 || ^0.12.20",
+        "phpstan/phpstan-phpunit": "^0.11.0 || ^0.12.16",
```

I don't think that part is necessary for PHPStan, is it?

```suggestion
        "phpstan/phpstan": "^0.12.20",
        "phpstan/phpstan-phpunit": "^0.12.16",
```

---

In order for composer to be able to resolve for 7.1 to 8.0 I found these changes necessary.

---

Are you sure? 0.12 is compatible with PHP 7.1 …

---

Yes but this library fails badly on 0.12 :(

---

Ok so it's locked to 0.11 in `composer.lock`, is that it?

---

Yep - I've added a comment to .travis.yml to this effect.